var/spack/repos/builtin/packages/py-numpy/package.py

```diff
@@ -74,9 +75,8 @@ class PyNumpy(PythonPackage):
     version("1.17.4", sha256="f58913e9227400f1395c7b800503ebfdb0772f1c33ff8cb4d6451c06cabdf316")
     version("1.17.3", sha256="a0678793096205a4d784bd99f32803ba8100f639cf3b932dc63b21621390ea7e")
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
```

Comment:
From what I understand, numpy does not require a Fortran compiler (although scipy does). These were automatically added simply because the f2py tests contain Fortran files.